src/main/resources/application.properties

```diff
@@ -7,8 +7,6 @@ azure.accessTokens.path=
 
 helium.keyvault.secretName=MySecret
 
-
-logging.level.com.microsoft.cse.helium.app=INFO
```

Comment:
Make sure we add a place to set a system default for logging in place of getting a command arg.

Reply:
@jofultz done. The default is now in the code or specifying on the cmdline. I haven't seen it done differently in other environments.